lib/pages/home.dart

```diff
@@ -1118,6 +1118,41 @@ double _carouselHeight(double scaleFactor, BuildContext context) => math.max(
         scaleFactor,
     _carouselHeightMin);
 
+typedef LibraryLoader = Future<void> Function();
```

Comment:
It might make more sense for the LibraryLoader and DeferredWidget to be moved into its own dart file

Reply:
thx for suggestion. Moved to deferred_widget.dart